app-utils/edid-decode/spec

```diff
@@ -0,0 +1,3 @@
+VER=0.1+git20230804
```

Comment:
Why the snapshot? Why not 0.1?

Reply:
It has no release tag.

Reply:
In that case, the portion before +git should be 0.